tests/Stl.Fusion.Tests/Services/EdgeCaseController.cs

```diff
@@ -39,4 +46,44 @@ public class EdgeCaseRewriteController : EdgeCaseController
     {
         public EdgeCaseRewriteController(IEdgeCaseService service) : base(service) { }
     }
+
+    #else
+    
+    [JsonifyErrors]
+    public class EdgeCaseController : ApiController, IEdgeCaseService
```

Comment:
Try to make sure most of controller code is the same.

Reply:
Done